pages/components/Home/OpenSourceHome.tsx

```diff
@@ -1,35 +1,25 @@
+import { Icon } from '@mui/material'
 import React from 'react'
 import ChallengeCardActive from '../cards/ChallengeCardActive'
 
+const Icons = [...Array(8)].map((_, i) => `icons/icon${i + 1}.svg`)
+console.log(Icons)
+
 const OpenSourceHome: React.FC = () => {
     return (
-        <div className='relative'>
-            <div className=' blur'>
-                <p className='sm:px-28 px-5 text-4xl mt-20 text-center md:text-left'>❤️ Start Contributing to Open Source </p>
-                <div className='flex p-28 items-center justify-between flex-col lg:flex-row'>
-                    <div className='lg:mr-10'>
-                        <img src="/Open-source Home.png" alt="" />
-                    </div>
-                    <div className='grid 2xl:grid-cols-2 gap-10 mt-20'>
-                        <ChallengeCardActive
-                            title="Web-development JS Challenge"
-                            xp={600}
-                            time='2jun-10jun'
-                            img="https://miro.medium.com/max/750/1*3T7J7csXY8u36acofw5N8g.jpeg"
-                        />
-                        <div className='block lg:hidden 2xl:block'>
-                            <ChallengeCardActive
-                                title="Web-development JS Challenge"
-                                xp={600}
-                                time='2jun-10jun'
-                                img="https://miro.medium.com/max/750/1*3T7J7csXY8u36acofw5N8g.jpeg" />
-                        </div>
-                    </div>
+        <div className='relative max-w-[1280px] mx-auto'>
```

Comment:
@Kingshuk3006 build a component for this use that ./pages/components/shared/TeachStackLine.tsx use that.

Use git fetch 
Then git pull origin stage

Then you van find it.